### ISSUE TITLE
using mkdfa the grammar compiler 

### ISSUE THREAD

**Issue by hiddenotebook** [julius-speech/julius#45]
Hi i am trying to use an application that I've used many times to control the music player, but when trying to compile grammar:

hiddenotebook@rules:~/julius/test-python$ mkdfa mediaplayer
The " mkdfa " program is not installed. You can install it by typing:
sudo apt install julius

This is not an error but I have Julius on /home/user/julius 
How I can redirect mkdfa to that directory?
I'm missing something?

I hope you can forgive me for the lack of experience. Thank you very much.

**Comment by colbec** [comment]
mkdfa is actually a perl script, so the call should be "mkdfa.pl ...". It should be installed during a normal Julius install. Try doing a "which mkdfa.pl" on your system. BTW the printed advice "apt install" is distro specific and probably should be revised.

**Comment by hiddenotebook** [comment]
Thank you very much for the explanation.

Wen I install julius with "apt install" mkdfa works fine, for that reason did not understand what was going on.

_BTW the printed advice "apt install" is distro specific and probably should be revised._

Yes sir right, but this is Ubuntu I'm just testing. what I want is to install julius on raspberry and there is no way install it from apt on raspbian jessie.

Thank you for your time I really appreciate it.

**Comment by colbec** [comment]
I have never played with a Raspberry Pi, but I do understand that the provided OSs can be rather pruned down to save space and therefore remove some functionality. In the case of Julius it should be possible to do all the development work of building grammars and language models on the big machine and then just transfer the results over to the small one. Important thing I guess would be to have audio samples in the audio model generated from the RPi.
Please keep us informed of your experiences with this small machine and Julius.

**Comment by hiddenotebook** [comment]
Good idea to safe some space. I'm learning how to programing but if you want to know how the thing goes I will be happy to report it. I love julius because don't need internet to do the recognition and is very fast. I'm making a wall-e robot with many sensors RGB leds on his eyes... I wish to make a good robot to take care my home and why not speak with him I use festival too for the speech synthesis. 

See you.

**Comment by hiddenotebook** [comment]
Hi colbec I left a video on youtube if you want to see how works julius on raspberry I was surprised how fast it works I love it.

https://youtu.be/4DocpEHsdi8
